micropub.php

```diff
@@ -279,7 +280,15 @@ private static function generate_args() {
     if (isset($_POST['url'])) {
       $args['ID'] = url_to_postid($_POST['url']);
     }
-
+    // perform these functions only for creates
+    if (!isset($args['ID'])) {
+      if (isset($args['post_title'])&&!isset($args['post_name'])) {
+        $args['post_name']=$args['post_title'];
+      }
+      if (isset($args['post_name'])) {
```

Comment:
we still want to sanitize for edits. it's just the defaulting that we want to limit to create.